### PR TITLE
Fix Admin football fronts

### PR DESCRIPTION
**Pull request by TBonnin**
## What does this change?

Fix for https://frontend.gutools.co.uk/admin/football/fronts "No data available" error

I am about to open a ticket with PA support to understand why some competitions like the uefa champion's league or fa cup, etc... returns 'no data' errors when requesting the participating teams list
## What is the value of this and can you measure success?

https://frontend.gutools.co.uk/admin/football/fronts works 😄 
## Does this affect other platforms - Amp, Apps, etc?

No
## Request for comment

@johnduffell @jfsoul 

Do not fail when PA client returns an exception when 'no data available'
error.

### CODE REVIEW

File: admin/app/football/controllers/FrontsController.scala [hunk 1]
```diff
@@ -27,7 +27,12 @@ class FrontsController(val wsClient: WSClient, val mode: Mode.Mode) extends Cont
   def index = Action.async { implicit request =>
     for {
       competitions <- client.competitions.map(PA.filterCompetitions)
-      competitionTeams <- Future.traverse(competitions){comp => client.teams(comp.competitionId, comp.startDate, comp.endDate)}
+      competitionTeams <- Future.traverse(competitions){ comp =>
+        client.teams(comp.competitionId, comp.startDate, comp.endDate).recover {
+          // 'No data' is returned as an error by PA API. Therefore we ignore exception and replace it by an empty list
+          case e: Exception => List()
```
Comment (jfsoul) [inline admin/app/football/controllers/FrontsController.scala]: Do we want to log/alert these exceptions somehow? I suppose we'll just get the heads up from central prod that there are competitions with empty lists anyway.

👍

**Comment by jfsoul** [comment]
👍  let's get the tool working, football season starts soon!

File: admin/app/football/controllers/FrontsController.scala (hunk 1 above)
Comment (johnduffell) [inline admin/app/football/controllers/FrontsController.scala]: which exception?

Comment (johnduffell) [inline admin/app/football/controllers/FrontsController.scala]: if you really want to swallow them at least log any that aren't the one we're expecting

Comment (TBonnin) [inline admin/app/football/controllers/FrontsController.scala]: You are right. let me limit those to the `PaClientException`

**Pull request merged**